cmd/agent/app/reporter/grpc/flags.go

```diff
@@ -22,13 +22,14 @@ import (
 )
 
 const (
-	gRPCPrefix             = "reporter.grpc."
-	collectorHostPort      = gRPCPrefix + "host-port"
-	retry                  = gRPCPrefix + "retry.max"
-	defaultMaxRetry        = 3
-	collectorTLS           = gRPCPrefix + "tls"
-	collectorTLSCA         = gRPCPrefix + "tls.ca"
-	collectorTLSServerName = gRPCPrefix + "tls.server-name"
+	gRPCPrefix                   = "reporter.grpc."
+	collectorHostPort            = gRPCPrefix + "host-port"
+	retry                        = gRPCPrefix + "retry.max"
+	defaultMaxRetry              = 3
+	collectorTLS                 = gRPCPrefix + "tls"
+	collectorTLSCA               = gRPCPrefix + "tls.ca"
+	collectorTLSServerName       = gRPCPrefix + "tls.server-name"
+	collectorConnectionsPerAgent = gRPCPrefix + "connections-per-host"
```

Comment:
where is "connections-per-host" name coming from? It's not what tchannel uses, we need to be consistent

Reply:
Updated